test/test_restful.py

```diff
@@ -108,13 +109,6 @@ def cmp(a, b):
 ek_tpm = None
 aik_tpm = None
 
-# Set up mTLS
-my_cert = config.get("tenant", "my_cert")
-my_priv_key = config.get("tenant", "private_key")
-cert = (my_cert, my_priv_key)
-tls_enabled = True
```

Comment:
Looks like this is still being used in further down in `test_035_test_delete_in_use_allowlist()`, which was added recently in #1088.

Reply:
Yes, just fixed this, will push again.